terasoluna-gfw-parent/pom.xml

```diff
@@ -539,7 +539,7 @@
     <!-- == TERASOLUNA == -->
     <terasoluna.gfw.version>5.7.3-SNAPSHOT</terasoluna.gfw.version>
     <!-- == Spring Boot == -->
-    <org.springframework.boot.version>2.7.7</org.springframework.boot.version>
+    <org.springframework.boot.version>2.7.12</org.springframework.boot.version>
     <!-- == MyBatis3 == -->
     <org.mybatis.version>3.5.11</org.mybatis.version>
     <org.mybatis.spring.version>2.1.0</org.mybatis.spring.version>
```

Comment:
- [ ] At the end it says `<cargo.tomcat9.version>9.0.73</cargo.tomcat9.version>`. Could you please change this to 9.0.75 to match the version of Tomcat used in Spring Boot 2.7.12?

Reply:
Please confirm the modification.